packages-exp/functions-compat/src/index.ts

```diff
@@ -18,19 +18,33 @@
 import firebase from '@firebase/app-compat';
 import { name, version } from '../package.json';
 import { registerFunctions } from './register';
-import * as types from '@firebase/functions-types';
+import { FirebaseFunctions as FunctionsCompat } from '@firebase/functions-types';
+import { Functions as FunctionsExp } from '@firebase/functions-exp';
 
+declare module '@firebase/functions-exp' {
```

Comment:
How about `getFunctions()`? We should define an overload that accepts a compat app instance.

Reply:
Done.